deployments/docker/standalone/docker-compose.yml

```diff
@@ -22,45 +20,74 @@ services:
     container_name: milvus-minio
     image: minio/minio:RELEASE.2023-03-20T20-16-18Z
     environment:
-      MINIO_ACCESS_KEY: minioadmin
-      MINIO_SECRET_KEY: minioadmin
+      MINIO_ROOT_USER: minioadmin
+      MINIO_ROOT_PASSWORD: minioadmin
     ports:
       - "9001:9001"
       - "9000:9000"
     volumes:
-      - ${DOCKER_VOLUME_DIRECTORY:-.}/volumes/minio:/minio_data
-    command: minio server /minio_data --console-address ":9001"
+      - minio-data:/data
+    command: server /data --console-address ":9001"
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
       interval: 30s
       timeout: 20s
       retries: 3
 
+  minio-init:
+    image: minio/mc:latest
+    depends_on:
+      - minio
+    entrypoint: >
+      /bin/sh -c "
+      until /usr/bin/mc config host add myminio http://minio:9000 minioadmin minioadmin; do
+        echo 'Waiting for MinIO to be available...'
+        sleep 1
+      done;
+      /usr/bin/mc mb myminio/milvus --ignore-existing;
+      /usr/bin/mc policy set public myminio/milvus;
+      echo 'MinIO initialized successfully'
+      "
+
   standalone:
     container_name: milvus-standalone
     image: milvusdb/milvus:v2.5.8
     command: ["milvus", "run", "standalone"]
     security_opt:
-    - seccomp:unconfined
+      - seccomp:unconfined
     environment:
-      MINIO_REGION: us-east-1
```

Comment:
we'd keep the MINIO_REGION env, and remove extra env like COMMON_STORAGETYPE, DATA_FLUSH_TIMEOUT, DATACOORD_ENABLECOMPACTION, DATACOORD_ENABLEGARBAGECOLLECTION, DATACOORD_SEGMENT_MAXSIZE

Reply:
Ok I have made these changes. You are correct the extra envs are a bit much.